sympy/parsing/latex/errors.py

```diff
@@ -0,0 +1,2 @@
+class LaTeXSyntaxError(Exception):
```

Comment:
`LaTeXParsingError` makes more sense to me, especially since many parsing errors will be from valid LaTeX.